src/dp_vnf.c

```diff
@@ -43,6 +43,24 @@ void dp_vnf_free(void)
 	dp_free_jhash_table(vnf_handle_tbl);
 }
 
+static __rte_always_inline
+void dp_fill_vnf_data(struct dp_vnf *vnf, enum dp_vnf_type type, uint16_t port_id, uint32_t vni,
+						const struct dp_ip_address *src, uint8_t prefix_len)
+{
+	memset(vnf, 0, sizeof(*vnf)); // set all bits to 0, including possible padding bits
+
+	vnf->type = type;
+	vnf->port_id = port_id;
+	vnf->vni = vni;
+	vnf->alias_pfx.length = prefix_len;
+	if (src->ip_type == RTE_ETHER_TYPE_IPV4) {
+		vnf->alias_pfx.ol.ip_type = src->ip_type;
+		vnf->alias_pfx.ol.ipv4 = src->ipv4;
+		return;
+	}
+	vnf->alias_pfx.ol = *src;
```

Comment:
I know this seems like a nitpick, but in this case I would rather use `if` and `else` as those branches are equal (one is not an error branch to warrant premature return).

Reply:
sure, done.